src/ThirdHomeWork.java

```diff
@@ -0,0 +1,83 @@
+public class ThirdHomeWork {
+    public static void main(String[] args) {
+        int[] a = {0, 2, 2, 3, 4, 4, 6};
+        int[] b = {1, 23, 278};
+
+        for (int i : mergeArrays(a, b)) {
+            System.out.print(i + " ");
+        }
+
+        String[] roles = {
+                "Городничий", "Аммос Федорович",
+                "Артемий Филиппович",
+                "Лука Лукич",
+                "Ревизор",
+                "Лука"};
+        String[] textLines = {"Городничий: Я пригласил вас, господа, с тем, чтобы сообщить вам пренеприятное известие: " +
+                "к нам едет ревизор.",
+                "Аммос Федорович: Как ревизор?",
+                "Артемий Филиппович: Как ревизор?",
+                "Городничий: Ревизор из Петербурга, инкогнито. И еще с секретным предписаньем.",
+                "Аммос Федорович: Вот те на!",
+                "Артемий Филиппович: Вот не было заботы, так подай!",
+                "Лука Лукич: Господи боже! еще и с секретным предписаньем!",
+                "Аммос Федорович: Вот это да-а-а!",
+                "Городничий: Городничий: Ревизор! Ревизор! Ревизор!",
+                "Лука: ... с секретным предписаньем!"};
+
+        System.out.print(printTextPerRole(roles, textLines));
+    }
+
+    public static int[] mergeArrays(int[] firstArray, int[] secondArray) {
+        int arrayLength = firstArray.length + secondArray.length;
+        int[] resultArray = new int[arrayLength];
+        int i = 0, j = 0, k = 0;
+
+        while (i < firstArray.length || j < secondArray.length) {
```

Comment:
optional: you can realize this method more simply

Reply:
Ok. I did it.